WinUIGallery/Samples/ControlPages/Design/TypographyPage.xaml

```diff
@@ -212,6 +214,21 @@
                             SizeLineHeight="14/20 epx"
                             VariableFont="Text, SemiBold"
                             Weight="600" />
+                        <controls:TypographyControl
```

Comment:
We should also add "Body Large" and "Body Large Strong" to *TypographySample_xaml.txt* for consistency.